internal/cli/atlas_clusters_list_test.go

```diff
@@ -41,8 +41,35 @@ func TestAtlasClustersList_Run(t *testing.T) {
 		Return(expected, nil).
 		Times(1)
 
-	err := listOpts.Run()
+	err := listOpts.RunAT()
 	if err != nil {
 		t.Fatalf("Run() unexpected error: %v", err)
 	}
 }
+
+func TestAtlasAllClustersProjectsList_Run(t *testing.T) {
```

Comment:
you don't need a new test function, you should just define a new scenario
this block should be able to live in the already existing function
```go
t.Run("List all clusters projects", func(t *testing.T) {...}
```